src/test/shell/bazel/bazel_symlink_test.sh

```diff
@@ -70,9 +70,25 @@ function set_up() {
   mkdir -p symlink
   touch symlink/BUILD
   cat > symlink/symlink.bzl <<EOF
```

Comment:
For whatever reason github won't let me add a comment near the top of the file -- on line 17 delete the word "unresolved" in the comment. Same for the `run_suite` call at the bottom of the file.